src/main/frontend/format/block.cljs

```diff
@@ -36,6 +38,34 @@ and handles unexpected failure."
   ([original-page-name with-id? with-timestamp?]
    (gp-block/page-name->map original-page-name with-id? (db/get-db (state/get-current-repo)) with-timestamp? (state/get-date-formatter))))
 
+(defn- block->string
+  "Convert block to string handling edge cases such as a, b, c being parsed as a set"
+  ([block]
+   (if (set? block) (string/join ", " block) (str block))))
```

Comment:
Comma separation of property values will no longer be the default behavior once #6529 lands. We should just use `(str)`